app/assets/stylesheets/home.scss

```diff
@@ -1,3 +1,84 @@
 // Place all the styles related to the home controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: https://sass-lang.com/
+
+
+a {
```

Comment:
What's the motivation for all these styles? Add a comment (or a few) in here to explain why you've added/changed these.

Reply:
Added comments and remove unused elements and css class selectors that were part of the bootstrap example I used.